docker/docker_client.go

```diff
@@ -51,7 +52,10 @@ const (
 	backoffMaxDelay      = 60 * time.Second
 )
 
-var systemPerHostCertDirPaths = [2]string{"/etc/containers/certs.d", "/etc/docker/certs.d"}
+var (
+	homeCertDir               = filepath.FromSlash(".config/containers/certs.d")
+	systemPerHostCertDirPaths = []string{filepath.Join(homedir.Get(), homeCertDir), "/etc/containers/certs.d", "/etc/docker/certs.d"}
```

Comment:
Evaluating `$HOME` at this point isn’t great — e.g. it makes it impossible to write unit tests with overridden $HOME.

Reply:
Ok moved the call out of init and just before it is used.